src/popup/components/Header.vue

```diff
@@ -40,7 +40,7 @@
       <BtnClose
         v-if="showHeaderNavigation"
         data-cy="close"
-        @click="close"
+        @click.capture="close"
```

Comment:
Instead of using `capture` I suggest to update the close button by removing the inner click event handler.